pkg/arvo/app/chat-hook.hoon

```diff
@@ -33,714 +32,69 @@
       invite-created=_|
       allow-history=(map path ?)
   ==
-::
-+$  poke
-  $%  [%chat-action action:store]
-      [%permission-action permission-action]
-      [%invite-action invite-action]
-      [%chat-view-action action:view]
```

Comment:
This file cannot be deleted because we do not have app tombstoning yet. It could instead be replaced with a stub that deletes its own state and kills all of its subscriptions. I'm also curious how you're handling the OTA case if this file has been deleted?

Reply:
Not sure what you're getting at here; the file *isn't* being deleted. I've basically done what you've asked: left a stub which deletes subscriptions. It doesn't kill the state because I need that around scryable so that when the new hooks start up, they can move over the state (e.g. the fact that you're subscribed to urbit-help or whatever).